uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/main/java/org/uberfire/ext/metadata/backend/infinispan/provider/InfinispanContext.java

```diff
@@ -278,17 +297,22 @@ public void dispose() {
     }
 
     public List<String> getIndices() {
-        return new ArrayList<>(this.cacheManager.getCacheNames());
+        return new ArrayList<>(this.cacheManager.getCacheNames())
+                .stream()
+                .filter(s -> s.startsWith(CACHE_PREFIX))
+                .map(s -> s.replace(CACHE_PREFIX,
```

Comment:
`s.replace(regularExression, replacement)` internally [compiles regular expression](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replaceAll(java.lang.String,%20java.lang.String)) every time you call it.
This could be made significantly faster by just using `.map(s -> s.substring(CACHE_PREFIX.length()))`